src/EditorFeatures/Core/Implementation/FindReferences/FindReferencesCommandHandler.cs

```diff
@@ -51,12 +52,15 @@ internal void FindReferences(ITextSnapshot snapshot, int caretPosition)
                     var service = document.Project.LanguageServices.GetService<IFindReferencesService>();
                     if (service != null)
                     {
-                        if (!service.TryFindReferences(document, caretPosition, context))
+                        using (Logger.LogBlock(FunctionId.CommandHandler_FindAllReference, context.CancellationToken))
```

Comment:
we can add such information if we want to. probably need to put that in the presenter though since find all reference engine is used on many different cases such as highlight reference, rename and etc. and sending that information on every highlight reference would be a problem.